f8a_worker/workers/mercator.py

```diff
@@ -193,7 +193,7 @@ def run_mercator(self, arguments, cache_path,
                        'details': []}
         mercator_target = arguments.get('cache_sources_path', cache_path)
         tc = TimedCommand(['mercator', mercator_target])
-        update_env = {'MERCATOR_JAVA_RESOLVE_POMS': 'true'} if resolve_poms else {}
```

Comment:
Yes, this does the trick, but `resolve_poms` argument becomes unused.

I'd rather see the change in [GraphAggregatorTask](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/workers/graphaggregator.py#L64), so that it runs the `MercatorTask` with `resolve_poms=True` (or not specifying `resolve_poms` at all so the default (`True`) value is used)

Reply:
Fair enough, update. PTAL.